dist/build_source_package.sh

```diff
@@ -92,6 +92,31 @@ build_srpm() {
     cp -v "${temp_dir}"/rpm/SRPMS/*.src.rpm "${output_dir}"/rpm/
 }
 
+build_pkgbuild() {
+    check_tools updpkgsums
+
+    mkdir -p "${temp_dir}"/pkg
+    sed \
+        -e "s/@VERSION@/${full_version}/g" \
```

Comment:
This I wasn't so sure about. For a true "git" package there would be a `ipmi-fan-control-git` package in AUR where we'd get the hash directly from the git checkout. 

In order to extract the tarball, which happens automatically during makepkg the version needed to match the full tarball name. 

I figured `VERSION_OVERRIDE` would be used in cases where we wouldn't want the git hash in the version name. 

Anyway, happy to tweak this as needed 

https://wiki.archlinux.org/index.php/PKGBUILD#version

Reply:
Agreed about the `ipmi-fan-control-git` package. I think we can leave it as you have it now.